data/human/news.txt

```diff
@@ -2150,3 +2150,164 @@ news "graffiti on spaceport wall"
 			"Veni, vidi, hyperjumped away."
 			"Those who dance will be called fools by those who cannot hear the music."
 			"Better half a sentence, "
+
+news "High Tension Situation"
+	location
+		attribute "ccor"
+name
+	"Standoff"
+message
+	word
+		"CCOR Guerrillas "
+		"Guerrillas in CCOR colors "
+	word
+		"are "
+	word
+		"distributing pampflets, "
+		"delivering aid, "
+		"rallying against the local authorities, "
+		"supporting local protests, "
+	word
+		"when "
+	word
+		"planetary authorities "
+		"local pirates "
+	word
+		"acost "
+		"confront "
+	word
+		"them to "
+	word
+		"dispurse. "
```

Comment:
disperse ? "To disperse is to scatter, and to disburse is to pay"

Reply:
Disperse is what I intended.

Reply:
```suggestion
		"disperse. "
```